e2e-polybft/e2e/consensus_test.go

```diff
@@ -624,7 +624,7 @@ func TestE2E_Consensus_MintableERC20NativeToken(t *testing.T) {
 		validatorCount,
 		framework.WithMintableNativeToken(true),
 		framework.WithEpochSize(epochSize),
-		framework.WithSecretsCallback(func(addrs []types.Address, config *framework.TestClusterConfig) {
+		framework.WithSecretsCallback(func(addrs []types.Address, config *framework.TestClusterConfig, tc *framework.TestCluster) {
```

Comment:
Seems like you can discard the `TestCluster` parameter in this and all the other tests in `consensus_test`.

```suggestion
		framework.WithSecretsCallback(func(addrs []types.Address, config *framework.TestClusterConfig, _ *framework.TestCluster) {
```

Reply:
fixed https://github.com/0xPolygon/polygon-edge/commit/3d92de70afa94706f3002144958fe9b5783ba13c